src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.Hibernate;
+
+@Getter
+@Setter
+@ToString

```

Comment:
In the future, be careful with this annotation on the `@Entity classes`. It may cause an N+1 problem or LazyInitializationException. We will learn about them later

Reply:
Okay. Got it.